src/ProductConstructionService/ProductConstructionService.Api/Queue/QueueConfiguration.cs

```diff
@@ -1,20 +1,29 @@
 ﻿// Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
+using Microsoft.Extensions.DependencyInjection;
+
 namespace ProductConstructionService.Api.Queue;
 
 public static class QueueConfiguration
 {
     public const string PcsJobQueueConfigurationKey = "PcsJobQueueName";
+    public const string PcsJobProcessorEmptyQueueWaitConfigurationKey = "EmptyQueueWaitSeconds";
```

Comment:
I would think twice before calling all the classes here with `Pcs*`. This should be clear from the namespace I'd say.

`BackgroundJob` or just `Job` might be fine.

Reply:
Yeah you're right, it's all in the namespace. I'll remove the Pcs everywhere